ray-operator/config/samples/ray-cluster.autoscaler.debug.yaml

```diff
@@ -0,0 +1,219 @@
+# This is adapted from https://github.com/ray-project/kuberay/blob/master/ray-operator/config/samples/ray-cluster.complete.yaml
+# It is a general RayCluster that has most fields in it for maximum flexibility in the Ray/Kuberay integration MVP.
+apiVersion: ray.io/v1alpha1
+kind: RayCluster
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+    # An unique identifier for the head node and workers of this cluster.
+  name: raycluster-complete
```

Comment:
can you please tell will this modifier be unique when a user launches multiple ray clusters?

Reply:
The names are required to be distinct for different Ray clusters --
It is up to the user (or if relevant, a higher-level controller) to use distinct names for Ray cluster CRs.